### PR TITLE
fix(ReportsPageRanking): show more pages fixed

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Show more pages when have more pages, before this pr only show one time.

image

### CODE REVIEW

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.test.js [hunk 1]
```diff
@@ -95,7 +98,7 @@ describe('Reports pages ranking', () => {
   it('should show empty message when dont have pages', async () => {
     const datahubClientDouble = {
       getPagesRankingByPeriod: async () => {
-        return { success: true, value: [] };
+        return { success: true, value: { hasMorePages: false, pages: [] } };
```
Comment (cbernat) [inline src/components/Reports/ReportsPageRanking/ReportsPageRanking.test.js]: You have a scenario for `hasMorePages` in `true` right?

File: src/services/datahub-client.test.ts [hunk 2]
```diff
@@ -378,20 +378,23 @@ describe('HttpDataHubClient', () => {
...
-          },
-          {
-            name: '/precios',
-            totalVisitors: 9000,
-            url: 'http://doppler.test/precios',
-            withEmail: 300,
-          },
-        ],
+        value: {
+          hasMorePages: false,
```
Comment (cbernat) [inline src/services/datahub-client.test.ts]: Perhaps you could add a scenario for `hasMorePages=true`

**Comment by cbernat** [comment]
👍

File: src/services/datahub-client.test.ts (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/services/datahub-client.test.ts]: yes! i added now one

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.test.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsPageRanking/ReportsPageRanking.test.js]: yes i added a test

**Comment by leoslopez** [comment]
👍

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.test.js [hunk 3]
```diff
@@ -111,4 +114,41 @@ describe('Reports pages ranking', () => {
...
+      <AppServicesProvider forcedServices={{ datahubClient: datahubClientDouble }}>
+        <DopplerIntlProvider>
+          <ReportsPageRanking domainName={domain} dateTo={fakeDate} dateFrom={fakeDate} />
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    expect(container.querySelector('.loading-box')).toBeInTheDocument();
+    await waitForDomChange();
+    expect(getByText('reports_pageranking.more_results'));
```
Comment (cbernat) [inline src/components/Reports/ReportsPageRanking/ReportsPageRanking.test.js]: ❤️

**Comment by leoslopez** [comment]
@fcoronelmakingsense, only to remember... It has already been deployed in production, so you can verify the correct behavior there.

**Pull request merged**